x/evidence/simulation/decoder.go

```diff
@@ -0,0 +1,34 @@
+package simulation
+
+import (
+	"bytes"
+	"fmt"
+
+	tmkv "github.com/tendermint/tendermint/libs/kv"
+
+	"github.com/cosmos/cosmos-sdk/x/evidence/types"
+)
+
+// DONTCOVER
```

Comment:
nit: we should ideally have a test case :-p

Reply:
removed `DONTCOVER `